sdk/python/packages/flet/src/flet/matplotlib_chart.py

```diff
@@ -1 +1 @@
-from flet.core.matplotlib_chart import MatplotlibChart
+
```

Comment:
This is a proxy to avoid the entire `flet` import failing if Matplotlib is not installed. Should be here.